spot-ml/src/main/scala/org/apache/spot/dns/DNSSuspiciousConnectsAnalysis.scala

```diff
@@ -68,14 +50,59 @@ object DNSSuspiciousConnectsAnalysis {
     logger.info("Scoring")
     val scoredDF = model.score(sparkContext, sqlContext, rawDataDF, userDomain)
```

Comment:
at one time, we had a mix of dataframes and RDDs, but now it's all dataframes, so there's no need to call that out with all these variable names ending in DF

names like scoredDNSRecords and rawDNSRecords would be more meaningful